openstf/pipeline/predict_sklearn.py

```diff
@@ -1,18 +1,19 @@
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 from pathlib import Path
 
 import joblib
 import pandas as pd
-
+import structlog
 
 from ktpbase.database import DataBase
```

Comment:
The pipeline should not depend on the database anymore, so this import should be removed I think.